compiler/pash_compiler.py

```diff
@@ -92,10 +93,13 @@ def compile_ir(ir_filename, compiled_script_file, args, compiler_config):
     ret = None
     try:
         ret = compile_optimize_output_script(ir_filename, compiled_script_file, args, compiler_config)
+    except UnparallelizableError as e: 
+        log("WARNING: Exception caught because some region(s) are unparallelizable:", e) 
+        log(traceback.format_exc()) # get detailed reports for (I) when debugging, one could simply uncomment/introduce traceback.print_exc() ? 
     except Exception as e:
         log("WARNING: Exception caught:", e)
-        # traceback.print_exc()
-
+        log(traceback.format_exc())
```

Comment:
Since this will now happen less often it might be OK to let the traceback be logged.